Source/MediaInfo/MediaInfo_Inform.cpp

```diff
@@ -511,12 +511,12 @@ Ztring MediaInfo_Internal::Inform()
                 Retour+=Inform((stream_t)StreamKind, StreamPos, false);
             }
 
-            if (HTML) Retour+=__T("</table>\n<br />");
+            if (HTML) Retour+=__T("</table>\n<br>");

```

Comment:
I have melancholia of XHTML :), I like to keep it (it should not harm anything, even the checkers).

Reply:
Don't mind this?
```
453.1-453.6: info: Trailing slash on void elements has no effect and interacts badly with unquoted attribute values.
```

Reply:
> Don't mind this?

I don't agree with the warning but a warning in a conformance checker has higher priority than my melancholia :).